src/program/lwaftr/doc/README.benchmarking.md

```diff
@@ -82,27 +83,72 @@ them to CPUs in the NUMA node 0.
 
 ![Decapsulation Gbps](benchmarks-v1.0/lwaftr-decapsulation-gbps.png)
 
+## Version 2.0
+
+Charts are not available at this moment.  Version 2.0 fixes packet loss
+for small binding tables, however there are still packet loss reported
+for big binding tables.  An excerpt of maximum peformance peak before 
+packets start to lose is pasted below.  See [loadtest.txt][benchmarks-v1.0/loadtest.txt]
+for a more detailed report.
+
+```
```

Comment:
We need a result with small binding tables too showing that we do in fact reach line rate.